packages/frontend/src/hooks/search/filters/index.ts

```diff
@@ -0,0 +1,32 @@
+import { ResolvedKPIToken } from "@carrot-kpi/sdk";
+
+export const filterTokensWithQuery = (
```

Comment:
This function is pretty pointless imho

Reply:
this is a nice abstraction, otherwise will have to do the looping on the useEffect


```js
const searchedTokens = filterTokensWithQuery(
    searchQuery,
    initialTokens
);
```

I thinks nicer this way.

Reply:
No but ser, it's literally pointless. What use have we for an additional function call (overhead) when the function is just a single instruction... there's no need to define one-line functions that are not called anywhere else, it doesn't add in clarity nor tidiness to the code (in fact, it removes it). If you want to make it clearer what's going on a comment is a better option.

Reply:
Ok, I buy this. 

I'll remove the abstraction.

Reply:
Have to agree, it's better now.

fixed: [6f8bc30](https://github.com/carrot-kpi/v1-monorepo/pull/234/commits/6f8bc30b9bcd66a3086016f2ef496fe641ff7286)